app/views/donations/edit.html.erb

```diff
@@ -0,0 +1,109 @@
+<% content_for :title, "Add to Donation #{@donation.id}" %>
+
+<% content_for :content do %>
+  <%= render partial: "donations/donation_row_template" %>
+  <%= render partial: "items/item_selector_template" %>
+
+  <%= javascript_tag do %>
+    var data = data || {};
+    data.categories = <%= Category.to_json.html_safe %>;
+
+    $.tableEditable("donation-table").initialize(1);
```

Comment:
```suggestion
    $.tableEditable("donation-table").initialize(0);
```

I noticed that when you access this page that you are either required to fill out item details or discard them you to be able to save. I have a feeling if your intention is to change just the date you shouldn't be required to operate on items at all.

![screen shot 2018-10-23 at 10 01 03 pm](https://user-images.githubusercontent.com/11335191/47401614-14032880-d710-11e8-9ca6-03241ed9b407.png)
 
This modification should make it so that the user explicitly has chosen to add an item.

Reply:
First and foremost thank you so much for your effort in reviewing this PR. @smellsblue and I have been extremely busy as of late and are finally returning to production and just getting around to your comments. My most sincere apologies, 

I will talk with our client about changing site-wide new blank row on creation. At present this has been our standard but we will discuss making this change and create a ticket if they agree.